python/pfs/instmodel/slit.py

```diff
@@ -59,13 +59,13 @@ class Slit(object):
 
     slits = slit1, slit2, slit1, slit2
 
-    def __init__(self, spectrograph):
+    def __init__(self, spectrograph, allHoles=False):
```

Comment:
Might be useful now to have a docString specifying the meaning of each of these parameters.